exercises/sublist/sublist_test.rb

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env ruby
+
+gem 'minitest', '>= 5.0.0'
+require 'minitest/autorun'
+require_relative 'sublist'
+
+class Sublist < Minitest::Test
```

Comment:
There is a JSON file with common tests that other languages have implemented for this problem, you can look it [here](https://github.com/exercism/x-common/blob/master/sublist.json) and create similar tests and expectations. :)